src/app/pages/RuntimeTransactionDetailPage/index.tsx

```diff
@@ -201,6 +224,66 @@ export const RuntimeTransactionDetailView: FC<{
             </>
           )}
 
+          {transfers && transfers.length > 0 && (
+            <>
+              <dt>{t('common.summary')}</dt>
+              <dd>
+                <Box sx={{ display: 'block', overflowX: 'auto' }}>
+                  {transfers.map((transfer, i) => {
+                    const params = transfer.evm_log_params
+                    if (!params) return null
+
+                    const from = params.find(p => p.name === 'from')?.value as string | undefined
+                    const to = params.find(p => p.name === 'to')?.value as string | undefined
+
+                    return (
+                      <Box
+                        key={i}
+                        sx={{
+                          display: 'flex',
+                          flexDirection: isMobile ? 'column' : 'row',
+                          alignItems: isMobile ? 'flex-start' : 'center',
+                          gap: isMobile ? 0 : 2,
+                          mb: isMobile ? 4 : 2,
+                          minWidth: '100%',
+                          whiteSpace: 'nowrap',
+                          lineHeight: '1.2',
+                        }}
+                      >
+                        <TokenTypeTag tokenType={transfer.evm_token?.type} />
+                        <Typography variant="body2">
+                          {t('common.from')}{' '}
+                          {from ? <AccountLink scope={transaction} address={from} alwaysTrim /> : '?'}
+                        </Typography>
+
+                        <Typography variant="body2">
+                          {t('common.to')}{' '}
+                          {to ? <AccountLink scope={transaction} address={to} alwaysTrim /> : '?'}
+                        </Typography>
+
+                        <Typography variant="body2">
+                          {t('common.for')} <EventBalance event={transfer} tickerAsLink={false} />
+                        </Typography>
+                      </Box>
+                    )
+                  })}
+                  {(totalTransfers ?? 0) > transfers.length && (
+                    <>
+                      <Link
+                        component={RouterLink}
+                        to={`${location.pathname}#${transactionEventsContainerId}`}
```

Comment:
I'll convert this to an absolute path, not based on current path. In search results this would incorrectly link to `/mainnet/sapphire/search#transactionEvents`